views/country_download.erb

```diff
@@ -123,77 +123,41 @@
 </div>
 <script>
 
-function get_q_param(key) {
+function getQueryParam(key) {
   var match = location.search.match(new RegExp("[?&]" + key + "=([^&]+)(&|$)"));
   return match && decodeURIComponent(match[1].replace(/\+/g, " "));
 }
 
-var max_visible_terms = 6;
-var reduced_visible_terms = 3; // best odd, so wanted term is in middle?
-var wanted_house = get_q_param('house');
-var wanted_term = get_q_param('term');
+var maxVisibleTerms = 6;
+var reducedVisibleTerms = 3; // best odd, so wanted term is in middle?
+var wantedHouse = getQueryParam('house');
+var wantedTerm = getQueryParam('term');
+var pageInitFunction = function(){
 
-// for each house...
+  // for each house...
+  // highlight the specific term (if there is one)
+  // collapse terms if there are many
```

Comment:
Now that the code has been factored out to elsewhere, you've ended up with a comment that simply repeats in English what the code below it is doing, which is almost always a bad smell. If the code still isn't sufficiently legible to make the comment obsolete, then there's still more refactoring to do. Or, if it is, you should drop the comment.